### DISCUSSION TITLE
update wording merit statement

### DISCUSSION THREAD

**Discussion by angaertner** [RESQUE-Framework/website#97] [Ideas]
adapt merit statement and include example, e.g. (by Medienpsychologie): 
A merit/ impact statement summarizes the contribution of the article to the field and allows to highlight its strengths on the examined criteria and beyond. For example, if the sample was specifically hard to recruit, the project took place in an interdisciplinary setting, a new theory was developed or tested for the first time, an experiment rejected or confirmed a long debated effect or a methodological set-up was specifically innovative.

**Comment by nicebread** [comment]
And mention that merit is understood completely independent from impact - merit is about the intrinsic qualities of research, even if nobody had the chance to recognize or cite it.

**Comment by nicebread** [comment]
And consider:

- letzte Frage "How well did the scheme (except your merit statement) cover the merits of this research output?" --> gar nicht - weil das wird ja auch gar nicht abgefragt.
- "made an important contribution to the field" trennen von "why it should be considered during the hiring / promotion process" --> getrennt abfragen.

**Comment by nicebread** [comment]
Give an example on the website?

**Reply by angaertner** [reply]
we discussed that a concrete example for a merit statement might be to specific, but that criteria could be listed on which candidates can comment in the merit statement (see first entry above). But if we want to give a concrete example, maybe give *multiple* examples to show how much the statements can vary?

**Comment by nicebread** [comment]
Add a summative merit statement in the "Author Tab"?
